packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -140,19 +107,16 @@ class FormatToolbar extends Component {
 	}
 
 	addLink() {
```

Comment:
Why not move these methods to `LinkContainer`?

Reply:
It should be now possible as I removed the usage of `setState` in both methods shared with `toolbarControls` variable. I will double check if this is expected on mobile and open a follow-up PR 👍